media-sound/frescobaldi/frescobaldi-3.1.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
```

Comment:
can you test with at least py37, even better py38 as well?

Reply:
It depends on dev-python/python-ly that currently only has py36 and no maintainer.

Reply:
It also depends on dev-python/python-poppler-qt5, which is py36 only. 
Both those dependencies should be safe to version bump with py37 support, since that's what upstream uses as their macosx default.